### DISCUSSION TITLE
Rights content?

### DISCUSSION THREAD

**Discussion by IgorDePaula** [livewire/livewire#4573] [4. General]
I'm a Brazilian. some people created a telegram group named LivewireBrasil. And now they anounced that will post content only company ther and any content by livewire in general is closed, proihibited, like they is owner of livewire. I dont like this injustice with who produces content to community.

the group is https://t.me/livewirebrasil

**Comment by joshhanley** [comment]
@IgorDePaula thanks for bringing it up! I will let Caleb know 🙂

**Comment by diogofm7** [comment]
> @IgorDePaula thanks for bringing it up! I will let Caleb know 🙂

![Screenshot_20220302-203650.png](https://user-images.githubusercontent.com/33474923/156467889-782c7e8b-a93e-455b-b910-b45423a2d5c5.png)

Now the group has been closed and no one can write.
